tools/expotools/src/commands/AndroidUnitTests.ts

```diff
@@ -0,0 +1,55 @@
+import spawnAsync from '@expo/spawn-async';
+import chalk from 'chalk';
+
+import * as Directories from '../Directories';
+import * as Packages from '../Packages';
+
+const ANDROID_DIR = Directories.getAndroidDir();
+
+const excludedInTests = [
+  "expo-module-template",
+  "expo-bluetooth",
+  "expo-notifications",
+  "expo-in-app-purchases",
+  "expo-updates"
+]
+
+async function action() {
+  const unimodules = await Packages.getListOfPackagesAsync();
+
+  function consoleErrorOutput(output: string, label: string, color: (string) => string): void {
```

Comment:
```suggestion
  function consoleErrorOutput(output: string, label: string, colorifyLine: (string) => string): void {
```
Callback should rather be named in function-like style 😉 (some verb 🤔)

Reply:
Color is both verb and noun :) But i see your point.